packages/pillarbox-playlist/src/lang/de.json

```diff
@@ -1,7 +1,9 @@
 {
-  "Next Item": "Nächstes Element",
+  "NextItem": "Nächstes element",
```

Comment:
`Next item` instead of `Next Item` is even better.

Reply:
There was an issue with the way we imported the language assets, this trick was mitigating at the time. The issue was solved here: #23 I'm rebasing this branch with main so that I can remove this part.